client/src/components/MapMarker/__tests__/__snapshots__/AccuracyHalo.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `null`;
```

Comment:
check your logic - both of these render to null which feels like a bug

Reply:
My spelling is just that bad

Reply:
I dont see it